packages/hebao_v1/contracts/lib/MathUint.sol

```diff
@@ -42,4 +42,22 @@ library MathUint
         c = a + b;
         require(c >= a, "ADD_OVERFLOW");
     }
+
+    function toUint64(uint v)
```

Comment:
Seems like we are doing this more and more (also in the protocol). It's of course super simple, but perhaps to be safe we can just switch to the Open zeppeling lib?
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/SafeCast.sol